draft-ietf-core-sid.md

```diff
@@ -1308,6 +1314,12 @@ Assignment of SIDs to YANG items can be automated, the recommended process to as
 5. The "dependency-revision" should reflect the revision numbers of each
    YANG module that the YANG module imports at the moment of the generation.
 
+When updating a YANG module that is in active use, the existing SID assignments are maintained.
+(In contrast, when evolving an early draft that has not yet been adopted by a community of developers, SID assignments are often better done from scratch after a revision.)
+If the name of a schema node changes, but the data remain structurally and semantically similar to what was previously available under an old name, the SID that was used for the old name MAY continue to be used for the new name.
+If the meaning of an item changes, a new SID MAY be assigned to it; this is particular useful to allow the new SID to identify the new structure or semantics of the item.
+Note that these decisions are at generally at the discretion of the YANG module author, who should decide if the benefits of a manual intervention are worth the deviation from automatic assignment, which may incur manual intervention.
+
```

Comment:
IMO the thing that matters is if the CBOR encoding changes.
E.g., leaf type changed from a string to a number, or a number to a union.
Then a new SID SHOULD (or MUST) be used to avoid breaking a decoder that expects
the CBOR encoding not to change

Reply:
I agree with Andy: I think we should add a sentence about encoding changes.